src/components/ui/Panel.jsx

```diff
@@ -191,10 +191,12 @@ class Panel extends React.Component {
     const { height } = this.state;
     const panelHeight = this.panelContentRef.current.offsetHeight;
     const values = {
-      'default': height - (fitContent
-        ? panelHeight + FIT_CONTENT_PADDING
-        : DEFAULT_SIZE),
-      'minimized': height - (fitContent
+      'default': height -
+        (fitContent.includes('default') &&
+        (panelHeight + FIT_CONTENT_PADDING <= DEFAULT_SIZE)
+          ? panelHeight + FIT_CONTENT_PADDING
+          : DEFAULT_SIZE),
+      'minimized': height - (fitContent.includes('minimized')
```

Comment:
Not sure if this function is called on desktop, but [`.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) is not available on IE11.

Reply:
Isn't it removed during build by the js transpiler ?

Reply:
I don't think so. The transpiler deals with the js syntax, but doesn't transform such method calls.